### PR TITLE
LP-54 Transaction list

### PR DESCRIPTION
**Pull request by fimak**
Transactions list render regarding transaction types.

Sorry for several things I did. I had to save some time for now:
1. I just take existing code to create a component Transaction.
2. hack of styles because they are different on 2 pages.
Thanks

### CODE REVIEW

**Comment by biswanathsahoo** [comment]
@fimak Please resolve the conflicts

File: src/Components/Dashboard/Transactions/Transaction.js [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+              className={styles.transaction__name}
+            >
+              {`# ${data.transaction_item_id}`}
+            </a>
+            <br />
+            {' from '}
+            <span
+              className={styles.transaction__name}
+            >
+              {data.counterparty[0].email}
```
Comment (biswanathsahoo) [inline src/Components/Dashboard/Transactions/Transaction.js]: Here check if email is there or not if not check for phone number. Please refer line no 101 - 103

File: src/Components/Dashboard/Transactions/Transaction.js [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+              className={styles.transaction__name}
+            >
+              {`# ${data.transaction_item_id}`}
+            </a>
+            <br />
+            {' to '}
+            <span
+              className={styles.transaction__name}
+            >
+              {data.counterparty[0].email}
```
Comment (biswanathsahoo) [inline src/Components/Dashboard/Transactions/Transaction.js]: Here also, check if email is there or not if not check for phone number. Please refer line no 101 - 103

**Review by biswanathsahoo — CHANGES_REQUESTED** [review]

File: src/Components/Dashboard/Transactions/Transaction.js (hunk 1 above)
Comment (fimak) [reply src/Components/Dashboard/Transactions/Transaction.js]: Fixed

File: src/Components/Dashboard/Transactions/Transaction.js (hunk 2 above)
Comment (fimak) [reply src/Components/Dashboard/Transactions/Transaction.js]: fixed

**Review by biswanathsahoo — APPROVED** [review]

**Pull request merged**